### PR TITLE
fix: Remove static context instance

### PR DESCRIPTION
**Pull request by sujithkanna**
Remove usage of static constant to avoid activity memory leak.
Instead, replace static context by "this" instance.

### CODE REVIEW

File: app/src/main/java/vn/mbm/phimp/me/PhimpMe.java [hunk 1]
```diff
@@ -798,37 +796,32 @@ public void onResume() {
...
+                    .replace(R.id.fragment_container, frag)
+                    .commit();
+            currentScreen = HomeScreenState.GALLERY;
+        } else {
+            AlertDialog.Builder alertbox = new AlertDialog.Builder(this);
+            alertbox.setMessage(getString(R.string.exit_message));
+            alertbox.setPositiveButton(getString(R.string.yes), new DialogInterface.OnClickListener() {
+                @Override
+                public void onClick(DialogInterface dialog, int which) {
+                    finish();
```
Comment (Abhi2424shek) [inline app/src/main/java/vn/mbm/phimp/me/PhimpMe.java]: Dont change unwanted things!

**Comment by Abhi2424shek** [comment]
Merge conflicts.

**Comment by Abhi2424shek** [comment]
@sujithkanna pls leave a reply.

**Comment by sujithkanna** [comment]
Actually we already closed it bro. Because of the improper pull request. So I was planning to update my repo and make those changes again and will send a pull request. Caught up in a work. Will update sooner. But this pull request is already closed. If not, then I am really sorry for that. Thank you :)

**Comment by Abhi2424shek** [comment]
No need to say sorry, I closed it now though.

**Pull request closed without merging**